server/main/src/main/resources/production.conf

```diff
@@ -0,0 +1,38 @@
+include "main.conf"
+
+akka {
+  loglevel = DEBUG
+
+  remote {
+    netty.tcp {
+      hostname = ${?APP_ADDR}
+      port = 2552
+    }
+  }
+
+  cluster {
+    seed-nodes = [ ${?SEED_NODES} ]
+    jvm-ports = [ ${akka.remote.netty.tcp.port} ]
+    retry = 10s
+    min-nr-of-members = 2
+  }
+
+  persistence {
+    journal.plugin = "cassandra-journal"
+    snapshot-store.plugin = "cassandra-snapshot-store"
+  }
+}
+
+cassandra-journal {
+  contact-points = [ ${?JOURNAL} ]
+}
+
+cassandra-snapshot-store {
+  contact-points = [ ${?SNAPSHOT} ]
+}
+
+kafka {
```

Comment:
I would try to preserve the config from main/application.conf

```
producer {
    message.send.max.retries = 3
    metadata.broker.list = ${?KAFKA_BROKER_LIST}
    connect.timeout.ms = 500
    producer.type = "async"
    topic = "test"
  }
```

At least producer.type and topic are desirable. Although the value is now set to "test" it is required so the broker knows where to publish. We will update that when the Kafka topics are agreed.